plugins/woocommerce-admin/client/products/fields/attributes/attributes.tsx

```diff
@@ -28,9 +29,33 @@ export const Attributes: React.FC< AttributesProps > = ( {
 
 	return (
 		<AttributeControl
-			attributeType="regular"
 			value={ attributes }
+			onAdd={ () => {
+				recordEvent( 'product_add_attributes_modal_add_button_click' );
+			} }
```

Comment:
Totally not for this PR, just thinking aloud...

Seeing these callbacks all over the place makes me want to come up with a more declarative way of hooking up Tracks events to UI events. Of course, would get more tricky when we want to pass in event props.

One can dream. 😄